src/main/kotlin/racingcar/domain/IOHandler.kt

```diff
@@ -0,0 +1,72 @@
+package racingcar.domain
+
+import camp.nextstep.edu.missionutils.Console
+import racingcar.data.Car
+
+class IOHandler(private val validator: Validator) {
+    fun getCarNames(): List<String> {
+        val input = getInput()
+        val names = splitNameFromInput(input)
+
+        for (name in names) {
+            require(validator.checkCarName(name)) {
+                NAME_LENGTH_IS_NOT_MATCHED
+            }
+        }
+
+        return names
+    }
+
+    fun getMoveCount(): Int {
+        val input = getInput()
+        val count = changeNumStringToInt(input)
+
+        require(validator.checkMoveCount(count)) {
+            NOT_ALLOWED_COUNT
+        }
+
+        return count
+    }
+
+    fun showLocation(cars: List<Car>) {
+        for (car in cars) {
+            show(car)
+        }
+    }
+
+    fun showWinner(winners: List<Car>) {
+        val sentence = winners.joinToString(SEPARATOR_FOR_WINNER) {
+            it.name
+        }
+        show("$SENTENCE_FOR_WINNER$sentence")
```

Comment:
[3] 가독성을 위해 아래와 같이 수정하고, SENTENCE_FOR_WINNER(최종 우승자 : ")의 끝에 있는 공백을 제거하는 건 어떨까요?
```suggestion
        show("$SENTENCE_FOR_WINNER $sentence")
```